server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -61,8 +77,45 @@ public TabletsMutator mutateTablets() {
     return new TabletsMutatorImpl(context);
   }
 
+  private void mutateRootGcCandidates(Consumer<RootGcCandidates> mutator) {
+    String zpath = context.getZooKeeperRoot() + ZROOT_TABLET_GC_CANDIDATES;
+    try {
+      context.getZooReaderWriter().mutate(zpath, new byte[0], ZooUtil.PUBLIC, currVal -> {
+        String currJson = new String(currVal, UTF_8);
+
```

Comment:
nit. There's a lot of extra whitespace in this code... a blank line between almost every statement. Seems a bit excessive, since I need to decrease my font size, or keep scrolling to see the larger context for the code.

Reply:
The white space made sense to me because the single lines of code were doing such different things.

Reply:
It's fine... it's just harder to fit on a page to take it all in. It's not important.